elastic/kibana
# Add require_alias event log when indexing documents

Fixes #93971

## To verify

1. Comment out the alias section when creating the initial event log index (https://github.com/elastic/kibana/blob/main/x-pack/plugins/event_log/server/es/init.ts#L208-L213)
2. Start Kibana and notice the `no such index [.kibana-event-log-8.3.0] and [require_alias] request flag is [true] and [.kibana-event-log-8.3.0] is not an alias` error logged when trying to log the startup event doc.
